<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>431</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>535</width>
    <height>431</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>main_icon.png</normaloff>main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All System Information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard Disk Information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CPU Informatiion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAM Information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Network Information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Process Information</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>/home/illya/Client/hardware-57-1105855.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <zorder>comboBox</zorder>
   <zorder>textEdit</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChange_settings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionChange_settings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
